### PR TITLE
Word Show page

### PR DESCRIPTION
**Pull request by apple31415**
Le done

### CODE REVIEW

File: src/main/frontend/src/components/WordShow.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React, {useState, useEffect } from 'react'
+import ReviewForm from './ReviewForm'
+
+const WordShow = (props) => {
+  let wordId = props.match.params.id
+  const [pet, setPet] = useState([])
+  const [newReview, setNewReview] = useState([])
+
+  useEffect(() => {
+   fetch('api/v1/${props.location.pathname}')
```
Comment (jsim0809) [inline src/main/frontend/src/components/WordShow.js]: I'm thinking this is going to be the URL. @MtnGreen confirm?
```suggestion
   fetch('api/v1/words/${props.location.pathname}')
```

File: src/main/frontend/src/components/WordShow.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  const handleWordClick = () => {
+    let formState = newReview === true ? false : true
+    setNewReview(formState)
+  }
+
+  return (
+    <div>
+      <h1>Words Words Words</h1>
+      <p>Word: {word.name}</p>
+      <p>Definition: {word.deffinition}</p>
```
Comment (jsim0809) [inline src/main/frontend/src/components/WordShow.js]:
```suggestion
      <p>Definition: {word.definition}</p>
```

File: src/main/frontend/src/components/WordShow.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+      <p>Language: {word.name_id}</p>
```
Comment (jsim0809) [inline src/main/frontend/src/components/WordShow.js]:
```suggestion
      <p>Language: {word.language.name}</p>
```

File: src/main/frontend/src/components/WordShow.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+const WordShow = (props) => {
+  let wordId = props.match.params.id
+  const [pet, setPet] = useState([])
+  const [newReview, setNewReview] = useState([])
+
+  useEffect(() => {
+   fetch('api/v1/${props.location.pathname}')
+    .then(result => result.json())
+    .then(word => {
+      setWord(word)
```
Comment (DanAlberts) [inline src/main/frontend/src/components/WordShow.js:13]: Here I see you use setWord, but I do not know where this function is on the page?

File: src/main/frontend/src/components/WordShow.js [hunk 5]
```diff
@@ -0,0 +1,31 @@
+import React, {useState, useEffect } from 'react'
+import ReviewForm from './ReviewForm'
+
+const WordShow = (props) => {
+  let wordId = props.match.params.id
+  const [pet, setPet] = useState([])
```
Comment (DanAlberts) [inline src/main/frontend/src/components/WordShow.js]: Are you reviewing pets?

**Review by DanAlberts — CHANGES_REQUESTED** [review]

**Review by jsim0809 — CHANGES_REQUESTED** [review]
A few small changes

File: src/main/frontend/src/components/WordShow.js (hunk 4 above)
Comment (DanAlberts) [reply src/main/frontend/src/components/WordShow.js:13]: This will need to be changed in order for it to work properly.

**Review by DanAlberts — APPROVED** [review]

File: src/main/frontend/src/components/WordShow.js (hunk 1 above)
Comment (jsim0809) [reply src/main/frontend/src/components/WordShow.js]: Nevermind, my bad!

**Pull request merged**